### PR TITLE
PR: Update conda install command for spyder-kernels 1.x

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Followed [PEP8](https://www.python.org/dev/peps/pep-0008/) for code style
* [x] Ensured your pull request hasn't eliminated unrelated blank lines/spaces,
      modified the ``spyder/defaults`` directory, or added new icons/assets
* [x] Described your changes and the motivation for them below

## Description of Changes

Per the discussion in spyder-ide/spyder-docs#69 , this PR corrects the install command for ``spyder-kernels`` 1.x to use ``conda-forge``, since it isn't available for all platforms on the main ``defaults`` channel. Also, fixes a few minor but silly typos in the lines near it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ corresponding `1.x` version of `spyder-kernels`.
 This can be done via two methods: installing the correct version via `conda`:
 
 ```bash
- conda install spyder-kernels=1
+conda install -c conda-forge spyder-kernels=1.*
```
Comment (ccordoba12) [inline CONTRIBUTING.md]: As I mentioned in the spyder-docs PR, please change this to

    conda install -c spyder-ide spyder-kernels=1.*

File: README.md [hunk 2]
```diff
@@ -149,14 +149,14 @@ python bootstrap.py
 
 You also need to make sure the correct `spyder-kernels` version is installed
 for the version of Spyder you are testing. The above procedure will give you
-`spyer-kernels` 0.x for the `3.x` branch (Spyder 3),
+`spyder-kernels` 0.x for the `3.x` branch (Spyder 3),
 so to run the `master` branch (Spyder 4) you need to additionally execute:
 
 ```bash
-conda install spyder-kernels=1.*
+conda install -c conda-forge spyder-kernels=1.*
```
Comment (ccordoba12) [inline README.md]: `conda-forge` -> `spyder-kernels`

File: CONTRIBUTING.md (hunk 1 above)
Comment (CAM-Gerlach) [reply CONTRIBUTING.md]: Yup

File: README.md (hunk 2 above)
Comment (CAM-Gerlach) [reply README.md]: :+1:

**Comment by ccordoba12** [comment]
Again, let's wait until we move the spyder-kernels packages to our channel.

**Comment by CAM-Gerlach** [comment]
Thanks!

**Pull request merged**